src/tests/ftest/container/basic_tx_test.yaml

```diff
@@ -7,13 +7,16 @@ hosts:
         - boro-B
 timeout: 100
 faults: !mux
-    two_faults:
-      fault_list:
-        - obj_update_timeout
-        - bogus_2nd_fault
-    just_one:
-      fault_list:
-        - obj_update_timeout
+# commenting out the fault injection bit
```

Comment:
Scott and I were talking about this the other day.   To aid in tracking of issues blocking tests we really should be using something easy to search for, like `skipForTicket()` or `self.cancelForTicket()`, in the python code rather than commenting out lines in the yaml file.  In this case we could call `self.cancelForTicket()` when the fault_list is defined.